zulip/integrations/git/post-receive

```diff
@@ -37,15 +40,26 @@ def git_repository_name() -> Text:
         return os.path.basename(os.path.dirname(os.getcwd()))
 
 def git_commit_range(oldrev: str, newrev: str) -> str:
+    remote_repo_cmd = ["git", "config", "--get", "remote.origin.url"]
+    remote_repo_url = subprocess.check_output(remote_repo_cmd).strip().decode("utf-8")
+
     log_cmd = ["git", "log", "--reverse",
-               "--pretty=%aE %H %s", "%s..%s" % (oldrev, newrev)]
+               "--pretty=%aN%n%H%n%h%n%s", "%s..%s" % (oldrev, newrev)]
     commits = ''
-    for ln in subprocess.check_output(log_cmd, universal_newlines=True).splitlines():
-        author_email, commit_id, subject  = ln.split(None, 2)
-        if hasattr(config, "format_commit_message"):
-            commits += config.format_commit_message(author_email, subject, commit_id)
-        else:
-            commits += '!avatar(%s) %s\n' % (author_email, subject)
+    output = subprocess.check_output(log_cmd, universal_newlines=True).splitlines()
+    it = iter(output)
+    for _ in range(len(output)//4):
+        author_name = next(iter(it))
+        commit_hash = next(iter(it))
+        commit_short_hash = next(iter(it))
+        subject = next(iter(it))
+        commits += COMMIT_ROW_TEMPLATE.format(
+            author_name=author_name,
+            commit_short_hash=commit_short_hash,
+            subject=subject,
+            commit_url="{}/commit/{}".format(remote_repo_url, commit_hash)
+        )
```

Comment:
For the formatting, I think it'd be better to actually match the format in https://chat.zulip.org/#narrow/stream/8-commits/topic/zulip.20.2F.20master/near/1121032.  Is there a reason you went with a different style?